builtin/plugin/backend.go

```diff
@@ -79,22 +79,44 @@ func Backend(ctx context.Context, conf *logical.BackendConfig) (*PluginBackend,
 	// Get SpecialPaths and BackendType
 	paths := raw.SpecialPaths()
 	btype := raw.Type()
+	runningVersion := ""
+	if versioner, ok := raw.(logical.Versioner); ok {
+		runningVersion = versioner.Version().Version
+	}
 
 	// Cleanup meta plugin backend
 	raw.Cleanup(ctx)
 
-	// Initialize b.Backend with dummy backend since plugin
+	// Initialize b.Backend with placeholder backend since plugin
 	// backends will need to be lazy loaded.
-	b.Backend = &framework.Backend{
-		PathsSpecial: paths,
-		BackendType:  btype,
+	b.Backend = &placeholderBackend{
```

Comment:
`placeholderBackend.IsExternal()` always returns true, but the function comment for `Backend` implies the backend could be builtin. Not sure if I'm just misreading though?

Reply:
Ah, you're right, the placeholder is used even for builtin plugins. I'll see if I can just forward the `IsExternal` method.